java-test/gate/python/PythonPRTest.java

```diff
@@ -117,24 +122,27 @@ public void testNullKeyInMap() throws Exception {
         controller.controllerExecutionFinished(controller);
 
         // Check that we got a new annotation in the document.
-        assertEquals(document.getAnnotations().size(), 13);
+        assertEquals(13, document.getAnnotations().size());
     }
 
 
+    // In GATE 8.4.1 and 8.6 test fails because
+    // some problem with default annotation set?
+    @Ignore
     @Test
     public void testAnnotationInDefaultSet() throws Exception {
         pythonPR.setScript(new File("examples/tokenise_badly.py").toURI().toURL());
 
         Document document = Factory.newDocument("anything goes");
 
-        document.getAnnotations().add(0l, 4l, "test", Factory.newFeatureMap());
```

Comment:
My understanding is that the Python script, executed below, is supposed to add an annotation (but it doesn't, which is probably a bug).

So I think this line only existed in order to get the test to pass.

Reply:
This whole test is a bug - there's two different `document` variables here, you're adding annotations to the local variable one but the python is running over the class field one.

Feel free to merge with this test disabled and I'll clean it up later.